src/pages/tasks/NewTaskDescriptionPage.js

```diff
@@ -70,7 +70,10 @@ const NewTaskDescriptionPage = (props) => {
         >
             <HeaderWithCloseButton
                 title={props.translate('newTaskPage.description')}
-                onCloseButtonPress={() => Navigation.dismissModal()}
+                onCloseButtonPress={() => {                    
+                    Navigation.dismissModal();
```

Comment:
Navigation.dismissModal() and TaskUtils.clearOutTaskInfo() is repeated at multiple places.

Can we create a util function like this?

https://github.com/Expensify/App/blob/465b78814589bffd88a1b90ccd66a7fc86b03900/src/libs/actions/Task.js#L479-L483

Reply:
@akinwale